app/views/poll_mailer/common/_footer.html.haml

```diff
@@ -1,7 +1,6 @@
 .poll-mailer-common-footer
   .poll-mailer-common-footer__spacer= PollMailer::REPLY_DELIMITER
   .poll-mailer-common-footer__text
-    -# %div= t(:"poll_mailer.common.why_send_this", reason: @info.send_reason)
-    = link_to t(:"poll_mailer.common.view_on_loomio", site_name: AppConfig.theme[:site_name]), @info.target_url
+    = link_to t(:"poll_mailer.common.view_on_loomio", site_name: AppConfig.theme[:site_name]), target_url({}, poll, recipient)
```

Comment:
maybe convert target_url to have optional args last... or all args are named params: eventable, recipient, args